Neos.CliSetup/Classes/Command/SetupCommandController.php

```diff
@@ -0,0 +1,193 @@
+    {
+        $availableDrivers = $this->databaseConnectionService->getAvailableDrivers();
+        if (count($availableDrivers) == 0) {
+            $this->outputLine("No supported database driver found");
+            $this->quit(1);
+        }
+
+        if (is_null($driver)) {
+            $driver = $this->output->select(
+                sprintf('DB Driver (<info>%s</info>): ', $this->persistenceConfiguration['driver'] ?? '---'),
+                $availableDrivers,
+                $this->persistenceConfiguration['driver']
+            );
+        }
+
+        if (is_null($host)) {
+            $host = $this->output->ask(
+                sprintf('Host (<info>%s</info>): ', $this->persistenceConfiguration['host'] ?? '127.0.0.1'),
+                $this->persistenceConfiguration['host'] ?? '127.0.0.1'
+            );
+        }
+
+        if (is_null($dbname)) {
+            $dbname = $this->output->ask(
+                sprintf('Database (<info>%s</info>): ', $this->persistenceConfiguration['dbname'] ?? '---'),
+                $this->persistenceConfiguration['dbname']
+            );
+        }
+
+        if (is_null($user)) {
+            $user = $this->output->ask(
+                sprintf('Username (<info>%s</info>): ', $this->persistenceConfiguration['user'] ?? '---'),
+                $this->persistenceConfiguration['user']
+            );
+        }
+
+        if (is_null($password)) {
+            $password = $this->output->ask(
+                sprintf('Password (<info>%s</info>): ', $this->persistenceConfiguration['password'] ?? '---'),
+                $this->persistenceConfiguration['password']
+            );
+        }
+
+        $persistenceConfiguration = [
+            'driver' => $driver,
+            'host' => $host,
+            'dbname' => $dbname,
+            'user' => $user,
+            'password' => $password
+        ];
+
+        try {
+            $this->databaseConnectionService->verifyDatabaseConnectionWorks($persistenceConfiguration);
+        } catch (SetupException $e) {
+            $this->outputLine($e->getMessage());
+            $this->quit(1);
+        }
+
+        $filename = $this->getSettingsFilename('Settings.Database.yaml');
+
+        $this->outputLine();
+        $this->output(sprintf('<info>%s</info>',$this->writeSettings($filename, 'Neos.Flow.persistence.backendOptions',$persistenceConfiguration)));
+        $this->outputLine();
+        $this->outputLine(sprintf("The new database settings were written to file <info>%s</info>", $filename));
+    }
+
+    /**
+     * @param string|null $driver
+     */
+    public function imageHandlerCommand(string $driver = null): void
+    {
+        $availableImageHandlers = $this->imageHandlerService->getAvailableImageHandlers();
+
+        if (count($availableImageHandlers) == 0) {
+            $this->outputLine("No supported image handler was found.");
```

Comment:
Suggestion: Either remove "was" here or add it to "No supported database driver .." text above